src/Base/Console.h

```diff
@@ -495,18 +495,18 @@ class BaseExport ConsoleSingleton
 {
 
 public:
-
+    static constexpr unsigned int BufferSize = 4024;

```

Comment:
@wwmayer 

Are you ok with this refactoring?

P.S.: Sorry for so many repetitions.